Build on FreeBSD 12.2 x86_64 works on 1.5, fails on 1.6 and master

Here is the full build log from building v1.6.0-rc2 from a fresh clone of the repository.

System info:
```
$ uname -a
FreeBSD freebsd-dilumtest12 12.2-RELEASE FreeBSD 12.2-RELEASE r366954 GENERIC  amd64
$ cc --version
FreeBSD clang version 10.0.1 (git@github.com:llvm/llvm-project.git llvmorg-10.0.1-0-gef32c611aa2)
Target: x86_64-unknown-freebsd12.2
Thread model: posix
InstalledDir: /usr/bin
```

<details><summary><code>autoreconf -i</code></summary>

```
$ autoreconf -i
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, 'config'.
libtoolize: copying file 'config/ltmain.sh'
libtoolize: Consider adding 'AC_CONFIG_MACRO_DIRS([m4])' to configure.ac,
libtoolize: and rerunning libtoolize and aclocal.
libtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
configure.ac:20: installing 'config/ar-lib'
configure.ac:16: installing 'config/compile'
configure.ac:10: installing 'config/config.guess'
configure.ac:10: installing 'config/config.sub'
configure.ac:11: installing 'config/install-sh'
configure.ac:11: installing 'config/missing'
Makefile.am: installing './INSTALL'
src/Makefile.am: installing 'config/depcomp'
parallel-tests: installing 'config/test-driver'
```

</details>

<details><summary><code>./configure</code></summary>

```
$ ./configure
checking build system type... amd64-unknown-freebsd12.2
checking host system type... amd64-unknown-freebsd12.2
checking target system type... amd64-unknown-freebsd12.2
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... config/install-sh -c -d
checking for gawk... no
checking for mawk... no
checking for nawk... nawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether to enable maintainer-specific portions of Makefiles... no
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of gcc... gcc3
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking dependency style of g++... gcc3
checking whether make sets $(MAKE)... (cached) yes
checking for ar... ar
checking the archiver (ar) interface... ar
checking how to print strings... printf
checking for a sed that does not truncate output... /usr/bin/sed
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for fgrep... /usr/bin/grep -F
checking for ld used by gcc... /usr/local/bin/ld
checking if the linker (/usr/local/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/local/bin/nm -B
checking the name lister (/usr/local/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 393216
checking how to convert amd64-unknown-freebsd12.2 file names to amd64-unknown-freebsd12.2 format... func_convert_file_noop
checking how to convert amd64-unknown-freebsd12.2 file names to toolchain format... func_convert_file_noop
checking for /usr/local/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for archiver @FILE support... no
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/local/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for mt... mt
checking if mt is a manifest tool... no
checking how to run the C preprocessor... gcc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... yes
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/usr/local/bin/ld) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... freebsd12.2 ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... no
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking how to run the C++ preprocessor... g++ -E
checking for ld used by g++... /usr/local/bin/ld
checking if the linker (/usr/local/bin/ld) is GNU ld... yes                                                                                           [225/1955]
checking whether the g++ linker (/usr/local/bin/ld) supports shared libraries... yes
checking for g++ option to produce PIC... -fPIC -DPIC
checking if g++ PIC flag -fPIC -DPIC works... yes
checking if g++ static flag -static works... yes
checking if g++ supports -c -o file.o... yes
checking if g++ supports -c -o file.o... (cached) yes
checking whether the g++ linker (/usr/local/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... freebsd12.2 ld.so
checking how to hardcode library paths into programs... immediate
checking dependency style of gcc... gcc3
checking for __uc_get_grs in -luca... no
checking for library containing dlopen... none required
checking for ANSI C header files... (cached) yes
checking asm/ptrace_offsets.h usability... no
checking asm/ptrace_offsets.h presence... no
checking for asm/ptrace_offsets.h... no
checking asm/ptrace.h usability... no
checking asm/ptrace.h presence... no
checking for asm/ptrace.h... no
checking endian.h usability... no
checking endian.h presence... no
checking for endian.h... no
checking sys/endian.h usability... yes
checking sys/endian.h presence... yes
checking for sys/endian.h... yes
checking sys/param.h usability... yes
checking sys/param.h presence... yes
checking for sys/param.h... yes
checking execinfo.h usability... yes
checking execinfo.h presence... yes
checking for execinfo.h... yes
checking ia64intrin.h usability... no
checking ia64intrin.h presence... no
checking for ia64intrin.h... no
checking sys/uc_access.h usability... no
checking sys/uc_access.h presence... no
checking for sys/uc_access.h... no
checking for unistd.h... (cached) yes
checking signal.h usability... yes
checking signal.h presence... yes
checking for signal.h... yes
checking for sys/types.h... (cached) yes
checking sys/procfs.h usability... yes
checking sys/procfs.h presence... yes
checking for sys/procfs.h... yes
checking sys/ptrace.h usability... yes
checking sys/ptrace.h presence... yes
checking for sys/ptrace.h... yes
checking sys/syscall.h usability... yes
checking sys/syscall.h presence... yes
checking for sys/syscall.h... yes
checking byteswap.h usability... no
checking byteswap.h presence... no
checking for byteswap.h... no
checking elf.h usability... yes
checking elf.h presence... yes
checking for elf.h... yes
checking sys/elf.h usability... yes
checking sys/elf.h presence... yes
checking for sys/elf.h... yes
checking link.h usability... yes
checking link.h presence... yes
checking for link.h... yes
checking sys/link.h usability... no
checking sys/link.h presence... no
checking for sys/link.h... no
checking for an ANSI C-conforming const... yes
checking for inline... inline
checking for size_t... yes
checking size of off_t... 8
checking for struct dl_phdr_info.dlpi_subs... yes
checking for struct elf_prstatus... no
checking for struct prstatus... yes
checking whether PTRACE_POKEUSER is declared... no
checking whether PTRACE_POKEDATA is declared... no
checking whether PTRACE_SETREGSET is declared... no
checking whether PTRACE_TRACEME is declared... no
checking whether PTRACE_CONT is declared... no
checking whether PTRACE_SINGLESTEP is declared... no
checking whether PTRACE_SYSCALL is declared... yes
checking whether PT_IO is declared... yes
checking whether PT_GETREGS is declared... yes
checking whether PT_GETFPREGS is declared... yes
checking whether PT_CONTINUE is declared... yes
checking whether PT_TRACE_ME is declared... yes
checking whether PT_STEP is declared... yes
checking whether PT_SYSCALL is declared... yes
checking for dl_iterate_phdr... yes
checking for dl_phdr_removals_counter... no
checking for dlmodinfo... no
checking for getunwind... no
checking for ttrace... no
checking for mincore... yes
checking for pipe2... yes
checking if building with AltiVec... no
checking for Android... no
checking if we should build libunwind-coredump... yes
checking for sys/ptrace.h... (cached) yes
checking if we should build libunwind-ptrace... yes
checking if we should export unwind.h... yes
checking if we should build libunwind-setjmp... yes
checking for build architecture... x86_64
checking for host architecture... x86_64
checking for target architecture... x86_64
checking for target operating system... freebsd12.2
checking for ELF helper width... 64
checking whether to include DWARF support... yes
checking whether to restrict build to remote support... no
checking whether to enable debug support...
checking whether to enable C++ exception support... no
checking whether to load .debug_frame sections... no
checking whether to block signals during mutex ops... yes
checking whether to validate memory addresses before use... yes
checking whether to enable msabi support...
checking whether to support LZMA-compressed symbol tables... auto
checking for lzma_mf_is_supported in -llzma... yes
checking whether to support ZLIB-compressed symbol tables... auto
checking for uncompress in -lz... yes
checking whether to support UNW_CACHE_PER_THREAD...
checking for Intel compiler... no
checking if building on Solaris then define __EXTENSIONS__ macro... yes
checking for QCC compiler... no
checking for __builtin___clear_cache... yes
checking for __builtin_unreachable... yes
checking for library containing backtrace... -lexecinfo
checking for latex2man... no
configure: WARNING: latex2man not found. Install latex2man. Disabling docs.
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating tests/Makefile
config.status: creating tests/check-namespace.sh
config.status: creating Makefile
config.status: creating src/Makefile
config.status: creating include/libunwind-common.h
config.status: creating include/libunwind.h
config.status: creating include/tdep/libunwind_i.h
config.status: creating src/unwind/libunwind.pc
config.status: creating src/coredump/libunwind-coredump.pc
config.status: creating src/ptrace/libunwind-ptrace.pc
config.status: creating src/setjmp/libunwind-setjmp.pc
config.status: creating src/libunwind-generic.pc
config.status: creating include/config.h
config.status: executing depfiles commands
config.status: executing libtool commands
```

</details>

<details><summary><code>gmake</code></summary>

```
$ gmake
Making all in src
gmake[1]: Entering directory '/usr/home/julia/Desktop/libunwind/src'
gmake  all-am
gmake[2]: Entering directory '/usr/home/julia/Desktop/libunwind/src'
depbase=`echo os-freebsd.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../include  -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG  -g -O2 -fe
xceptions -Wall -Wsign-compare -D__EXTENSIONS__ -MT os-freebsd.lo -MD -MP -MF $depbase.Tpo -c -o os-freebsd.lo os-freebsd.c &&\
mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT os-freebsd.lo -MD -MP -MF .deps/os-freebsd.Tpo -c os-freebsd.c  -fPIC -DPIC -o .libs/os-freebsd.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT os-freebsd.lo -MD -MP -MF .deps/os-freebsd.Tpo -c os-freebsd.c -o os-freebsd.o >/dev/null 2>&1
depbase=`echo mi/init.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../include  -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG  -g -O2 -fe
xceptions -Wall -Wsign-compare -D__EXTENSIONS__ -MT mi/init.lo -MD -MP -MF $depbase.Tpo -c -o mi/init.lo mi/init.c &&\
mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT mi/init.lo -MD -MP -MF mi/.deps/init.Tpo -c mi/init.c  -fPIC -DPIC -o mi/.libs/init.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT mi/init.lo -MD -MP -MF mi/.deps/init.Tpo -c mi/init.c -o mi/init.o >/dev/null 2>&1
depbase=`echo mi/flush_cache.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../include  -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG  -g -O2 -fe
xceptions -Wall -Wsign-compare -D__EXTENSIONS__ -MT mi/flush_cache.lo -MD -MP -MF $depbase.Tpo -c -o mi/flush_cache.lo mi/flush_cache.c &&\
mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT mi/flush_cache.lo -MD -MP -MF mi/.deps/flush_cache.Tpo -c mi/flush_cache.c  -fPIC -DPIC -o mi/.libs/flush_cache.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT mi/flush_cache.lo -MD -MP -MF mi/.deps/flush_cache.Tpo -c mi/flush_cache.c -o mi/flush_cache.o >/dev/null 2>&1
depbase=`echo mi/mempool.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../include  -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG  -g -O2 -fe
xceptions -Wall -Wsign-compare -D__EXTENSIONS__ -MT mi/mempool.lo -MD -MP -MF $depbase.Tpo -c -o mi/mempool.lo mi/mempool.c &&\
mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT mi/mempool.lo -MD -MP -MF mi/.deps/mempool.Tpo -c mi/mempool.c  -fPIC -DPIC -o mi/.libs/mempool.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT mi/mempool.lo -MD -MP -MF mi/.deps/mempool.Tpo -c mi/mempool.c -o mi/mempool.o >/dev/null 2>&1
depbase=`echo mi/strerror.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../include  -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG  -g -O2 -fe
xceptions -Wall -Wsign-compare -D__EXTENSIONS__ -MT mi/strerror.lo -MD -MP -MF $depbase.Tpo -c -o mi/strerror.lo mi/strerror.c &&\
mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT mi/strerror.lo -MD -MP -MF mi/.deps/strerror.Tpo -c mi/strerror.c  -fPIC -DPIC -o mi/.libs/strerror.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT mi/strerror.lo -MD -MP -MF mi/.deps/strerror.Tpo -c mi/strerror.c -o mi/strerror.o >/dev/null 2>&1
depbase=`echo x86_64/is_fpreg.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../include  -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG  -g -O2 -fe
xceptions -Wall -Wsign-compare -D__EXTENSIONS__ -MT x86_64/is_fpreg.lo -MD -MP -MF $depbase.Tpo -c -o x86_64/is_fpreg.lo x86_64/is_fpreg.c &&\
mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT x86_64/is_fpreg.lo -MD -MP -MF x86_64/.deps/is_fpreg.Tpo -c x86_64/is_fpreg.c  -fPIC -DPIC -o x86_64/.libs/is_fpreg.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT x86_64/is_fpreg.lo -MD -MP -MF x86_64/.deps/is_fpreg.Tpo -c x86_64/is_fpreg.c -o x86_64/is_fpreg.o >/dev/null 2>&1
depbase=`echo x86_64/regname.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../include  -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG  -g -O2 -fe
xceptions -Wall -Wsign-compare -D__EXTENSIONS__ -MT x86_64/regname.lo -MD -MP -MF $depbase.Tpo -c -o x86_64/regname.lo x86_64/regname.c &&\
mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT x86_64/regname.lo -MD -MP -MF x86_64/.deps/regname.Tpo -c x86_64/regname.c  -fPIC -DPIC -o x86_64/.libs/regname.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT x86_64/regname.lo -MD -MP -MF x86_64/.deps/regname.Tpo -c x86_64/regname.c -o x86_64/regname.o >/dev/null 2>&1
depbase=`echo x86_64/Los-freebsd.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../include  -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG  -g -O2 -fe
xceptions -Wall -Wsign-compare -D__EXTENSIONS__ -MT x86_64/Los-freebsd.lo -MD -MP -MF $depbase.Tpo -c -o x86_64/Los-freebsd.lo x86_64/Los-freebsd.c &&\
mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include/tdep-x86_64 -I. -D_GNU_SOURCE -DNDEBUG -g -O2 -fexceptions -Wall -Wsign-compar
e -D__EXTENSIONS__ -MT x86_64/Los-freebsd.lo -MD -MP -MF x86_64/.deps/Los-freebsd.Tpo -c x86_64/Los-freebsd.c  -fPIC -DPIC -o x86_64/.libs/Los-freebsd.o
In file included from x86_64/Los-freebsd.c:4:
x86_64/Gos-freebsd.c: In function '_ULx86_64_sigreturn':
x86_64/Gos-freebsd.c:195:28: error: 'struct cursor' has no member named 'uc'
  195 |   uc->uc_mcontext.mc_r8 = c->uc->uc_mcontext.mc_r8;
      |                            ^~
x86_64/Gos-freebsd.c:196:28: error: 'struct cursor' has no member named 'uc'
  196 |   uc->uc_mcontext.mc_r9 = c->uc->uc_mcontext.mc_r9;
      |                            ^~
x86_64/Gos-freebsd.c:197:29: error: 'struct cursor' has no member named 'uc'
  197 |   uc->uc_mcontext.mc_r10 = c->uc->uc_mcontext.mc_r10;
      |                             ^~
x86_64/Gos-freebsd.c:198:29: error: 'struct cursor' has no member named 'uc'
  198 |   uc->uc_mcontext.mc_r11 = c->uc->uc_mcontext.mc_r11;
      |                             ^~
x86_64/Gos-freebsd.c:199:29: error: 'struct cursor' has no member named 'uc'
  199 |   uc->uc_mcontext.mc_r12 = c->uc->uc_mcontext.mc_r12;
      |                             ^~
x86_64/Gos-freebsd.c:200:29: error: 'struct cursor' has no member named 'uc'
  200 |   uc->uc_mcontext.mc_r13 = c->uc->uc_mcontext.mc_r13;
      |                             ^~
x86_64/Gos-freebsd.c:201:29: error: 'struct cursor' has no member named 'uc'
  201 |   uc->uc_mcontext.mc_r14 = c->uc->uc_mcontext.mc_r14;
      |                             ^~
x86_64/Gos-freebsd.c:202:29: error: 'struct cursor' has no member named 'uc'
  202 |   uc->uc_mcontext.mc_r15 = c->uc->uc_mcontext.mc_r15;
      |                             ^~
x86_64/Gos-freebsd.c:203:29: error: 'struct cursor' has no member named 'uc'
  203 |   uc->uc_mcontext.mc_rdi = c->uc->uc_mcontext.mc_rdi;
      |                             ^~
x86_64/Gos-freebsd.c:204:29: error: 'struct cursor' has no member named 'uc'
  204 |   uc->uc_mcontext.mc_rsi = c->uc->uc_mcontext.mc_rsi;
      |                             ^~
x86_64/Gos-freebsd.c:205:29: error: 'struct cursor' has no member named 'uc'
  205 |   uc->uc_mcontext.mc_rbp = c->uc->uc_mcontext.mc_rbp;
      |                             ^~
x86_64/Gos-freebsd.c:206:29: error: 'struct cursor' has no member named 'uc'
  206 |   uc->uc_mcontext.mc_rbx = c->uc->uc_mcontext.mc_rbx;
      |                             ^~
x86_64/Gos-freebsd.c:207:29: error: 'struct cursor' has no member named 'uc'
  207 |   uc->uc_mcontext.mc_rdx = c->uc->uc_mcontext.mc_rdx;
      |                             ^~
x86_64/Gos-freebsd.c:208:29: error: 'struct cursor' has no member named 'uc'
  208 |   uc->uc_mcontext.mc_rax = c->uc->uc_mcontext.mc_rax;
      |                             ^~
x86_64/Gos-freebsd.c:209:29: error: 'struct cursor' has no member named 'uc'
  209 |   uc->uc_mcontext.mc_rcx = c->uc->uc_mcontext.mc_rcx;
      |                             ^~
x86_64/Gos-freebsd.c:210:29: error: 'struct cursor' has no member named 'uc'
  210 |   uc->uc_mcontext.mc_rsp = c->uc->uc_mcontext.mc_rsp;
      |                             ^~
x86_64/Gos-freebsd.c:211:29: error: 'struct cursor' has no member named 'uc'
  211 |   uc->uc_mcontext.mc_rip = c->uc->uc_mcontext.mc_rip;
      |                             ^~
gmake[2]: *** [Makefile:4113: x86_64/Los-freebsd.lo] Error 1
gmake[2]: Leaving directory '/usr/home/julia/Desktop/libunwind/src'
gmake[1]: *** [Makefile:2318: all] Error 2
gmake[1]: Leaving directory '/usr/home/julia/Desktop/libunwind/src'
gmake: *** [Makefile:610: all-recursive] Error 1
```

</details>

As an aside, Travis now supports FreeBSD, so it could be added to CI.

--- a/src/x86_64/Gos-freebsd.c
+++ b/src/x86_64/Gos-freebsd.c
@@ -192,23 +192,23 @@ x86_64_sigreturn (unw_cursor_t *cursor)
   ucontext_t *uc = (ucontext_t *)(c->sigcontext_addr +
     offsetof(struct sigframe, sf_uc));
 
-  uc->uc_mcontext.mc_r8 = c->uc->uc_mcontext.mc_r8;
-  uc->uc_mcontext.mc_r9 = c->uc->uc_mcontext.mc_r9;
-  uc->uc_mcontext.mc_r10 = c->uc->uc_mcontext.mc_r10;
-  uc->uc_mcontext.mc_r11 = c->uc->uc_mcontext.mc_r11;
-  uc->uc_mcontext.mc_r12 = c->uc->uc_mcontext.mc_r12;
-  uc->uc_mcontext.mc_r13 = c->uc->uc_mcontext.mc_r13;
-  uc->uc_mcontext.mc_r14 = c->uc->uc_mcontext.mc_r14;
-  uc->uc_mcontext.mc_r15 = c->uc->uc_mcontext.mc_r15;
-  uc->uc_mcontext.mc_rdi = c->uc->uc_mcontext.mc_rdi;
-  uc->uc_mcontext.mc_rsi = c->uc->uc_mcontext.mc_rsi;
-  uc->uc_mcontext.mc_rbp = c->uc->uc_mcontext.mc_rbp;
-  uc->uc_mcontext.mc_rbx = c->uc->uc_mcontext.mc_rbx;
-  uc->uc_mcontext.mc_rdx = c->uc->uc_mcontext.mc_rdx;
-  uc->uc_mcontext.mc_rax = c->uc->uc_mcontext.mc_rax;
-  uc->uc_mcontext.mc_rcx = c->uc->uc_mcontext.mc_rcx;
-  uc->uc_mcontext.mc_rsp = c->uc->uc_mcontext.mc_rsp;
-  uc->uc_mcontext.mc_rip = c->uc->uc_mcontext.mc_rip;
+  uc->uc_mcontext.mc_r8 = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_r8;
+  uc->uc_mcontext.mc_r9 = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_r9;
+  uc->uc_mcontext.mc_r10 = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_r10;
+  uc->uc_mcontext.mc_r11 = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_r11;
+  uc->uc_mcontext.mc_r12 = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_r12;
+  uc->uc_mcontext.mc_r13 = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_r13;
+  uc->uc_mcontext.mc_r14 = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_r14;
+  uc->uc_mcontext.mc_r15 = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_r15;
+  uc->uc_mcontext.mc_rdi = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_rdi;
+  uc->uc_mcontext.mc_rsi = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_rsi;
+  uc->uc_mcontext.mc_rbp = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_rbp;
+  uc->uc_mcontext.mc_rbx = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_rbx;
+  uc->uc_mcontext.mc_rdx = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_rdx;
+  uc->uc_mcontext.mc_rax = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_rax;
+  uc->uc_mcontext.mc_rcx = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_rcx;
+  uc->uc_mcontext.mc_rsp = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_rsp;
+  uc->uc_mcontext.mc_rip = dwarf_get_uc(&c->dwarf)->uc_mcontext.mc_rip;
 
   Debug (8, "resuming at ip=%llx via sigreturn(%p)\n",
              (unsigned long long) c->dwarf.ip, uc);

--- a/tests/test-ptrace.c
+++ b/tests/test-ptrace.c
@@ -233,7 +233,11 @@ main (int argc, char **argv)
         fprintf(stderr, "Need to specify a command line for the child\n");
         exit (-1);
       }
+#ifdef __FreeBSD__
+      execve (argv[optind], argv + optind, environ);
+#else
       execvpe (argv[optind], argv + optind, environ);
+#endif
       _exit (-1);
     }
   atexit (target_pid_kill);

